templates/engage/ai-ml-ubuntu.html

```diff
@@ -0,0 +1,52 @@
+{% extends "engage/base_engage.html" %}
+
+{% block meta_description %}Ready to make the migration from proprietary virtualisation to OpenStack?{% endblock %}
+
+{% block title %}AI, ML and Ubuntu: Everything you need to know{% endblock %}
+
```

Comment:
Can you add a copydoc metadata link here?
`{% block meta_copydoc %}https://docs.google.com/document/d/1YVPlQYT3jvxb6Yp5OShSiUsmKBPOYOm_fNLJHNsQ6tQ/edit{% endblock meta_copydoc %}`

Reply:
Done!